opentracing-spring-cloud-starter/Makefile

```diff
@@ -0,0 +1,8 @@
+
+PROFILES = nodeps feign
+
+ALL:
+	@for profile in $(PROFILES) ; do \
+		echo "---> Executing profile $$profile"; \
+		../mvnw clean test -P $$profile; \
```

Comment:
`../` ? Isn't it on the current directory (`./`)?

Reply:
makefile is in sub-directory.

Reply:
That's confusing. If it's starting a build on the parent, why isn't it at the parent's level?

Reply:
it's just reusing the mvnw from parent dir.

Reply:
Oh! It's using the Maven Wrapper from the parent to build the child. Ok, I see now.

Reply:
Is the makefile still necessary if only two profiles?